suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -561,7 +562,8 @@ export const createDescriptorPreloadedDiscoveryThunk = createThunk(
                 ({ accountType }) =>
                     // Some cardano derivation may not be supported by the device. We need to exclude them from total count.
                     network.networkType !== 'cardano' ||
-                    (availableCardanoDerivations as string[]).includes(accountType),
+                    (availableCardanoDerivations !== undefined &&
+                        isArrayMember(accountType, availableCardanoDerivations)),
```

Comment:
This was actually a potential runtime error caused by the `as`, at least as far as TS is concerned